2022-01-20 18:57:58,281 INFO    Thread-2  :224864 [wandb_setup.py:_flush():71] setting env: {'entity': 'amazonproject', 'project': 'Amazon-Reviews-v2', 'root_dir': '/zhome/4a/f/117237/Documents/MLOps_Amazon_review_project', 'sweep_id': 'igrmn63w', 'run_id': 'xpfo0wdl', 'sweep_param_path': '/zhome/4a/f/117237/Documents/MLOps_Amazon_review_project/wandb/sweep-igrmn63w/config-xpfo0wdl.yaml'}
2022-01-20 18:57:58,281 INFO    Thread-2  :224864 [wandb_setup.py:_flush():71] setting login settings: {}
2022-01-20 18:57:58,281 INFO    Thread-2  :224864 [wandb_init.py:_log_setup():371] Logging user logs to /zhome/4a/f/117237/Documents/MLOps_Amazon_review_project/wandb/run-20220120_185758-xpfo0wdl/logs/debug.log
2022-01-20 18:57:58,282 INFO    Thread-2  :224864 [wandb_init.py:_log_setup():372] Logging internal logs to /zhome/4a/f/117237/Documents/MLOps_Amazon_review_project/wandb/run-20220120_185758-xpfo0wdl/logs/debug-internal.log
2022-01-20 18:57:58,282 INFO    Thread-2  :224864 [wandb_init.py:init():404] calling init triggers
2022-01-20 18:57:58,282 INFO    Thread-2  :224864 [wandb_init.py:init():411] wandb.init called with sweep_config: {'batch_size': 100, 'drop_out': 0.25, 'lr': 0.001, 'optimizer': 'adam'}
config: {}
2022-01-20 18:57:58,282 INFO    Thread-2  :224864 [wandb_init.py:init():460] starting backend
2022-01-20 18:57:58,282 INFO    Thread-2  :224864 [backend.py:_multiprocessing_setup():101] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-01-20 18:57:58,296 INFO    Thread-2  :224864 [backend.py:ensure_launched():216] starting backend process...
2022-01-20 18:57:58,308 INFO    Thread-2  :224864 [backend.py:ensure_launched():222] started backend process with pid: 224890
2022-01-20 18:57:58,320 INFO    Thread-2  :224864 [wandb_init.py:init():469] backend started and connected
2022-01-20 18:57:58,322 INFO    Thread-2  :224864 [wandb_run.py:_config_callback():956] config_cb None None {'batch_size': 100, 'drop_out': 0.25, 'lr': 0.001, 'optimizer': 'adam'}
2022-01-20 18:57:58,324 INFO    Thread-2  :224864 [wandb_init.py:init():533] updated telemetry
2022-01-20 18:57:58,365 INFO    Thread-2  :224864 [wandb_init.py:init():563] communicating current version
2022-01-20 18:57:59,838 INFO    Thread-2  :224864 [wandb_init.py:init():568] got version response 
2022-01-20 18:57:59,838 INFO    Thread-2  :224864 [wandb_init.py:init():578] communicating run to backend with 30 second timeout
2022-01-20 18:58:00,081 INFO    Thread-2  :224864 [wandb_init.py:init():606] starting run threads in backend
2022-01-20 18:58:00,159 INFO    Thread-2  :224864 [wandb_run.py:_console_start():1810] atexit reg
2022-01-20 18:58:00,162 INFO    Thread-2  :224864 [wandb_run.py:_redirect():1684] redirect: SettingsConsole.REDIRECT
2022-01-20 18:58:00,162 INFO    Thread-2  :224864 [wandb_run.py:_redirect():1689] Redirecting console.
2022-01-20 18:58:00,163 ERROR   Thread-2  :224864 [wandb_run.py:_redirect():1748] Failed to redirect.
Traceback (most recent call last):
  File "/zhome/4a/f/117237/.local/lib/python3.6/site-packages/wandb/sdk/wandb_run.py", line 1741, in _redirect
    out_redir.install()
  File "/zhome/4a/f/117237/.local/lib/python3.6/site-packages/wandb/sdk/lib/redirect.py", line 709, in install
    _WSCH.add_fd(self._pipe_read_fd)
  File "/zhome/4a/f/117237/.local/lib/python3.6/site-packages/wandb/sdk/lib/redirect.py", line 659, in add_fd
    self._register()
  File "/zhome/4a/f/117237/.local/lib/python3.6/site-packages/wandb/sdk/lib/redirect.py", line 644, in _register
    old_handler = signal.signal(signal.SIGWINCH, lambda *_: None)
  File "/usr/lib64/python3.6/signal.py", line 47, in signal
    handler = _signal.signal(_enum_to_int(signalnum), _enum_to_int(handler))
ValueError: signal only works in main thread
2022-01-20 18:58:00,166 INFO    Thread-2  :224864 [wandb_init.py:init():633] run started, returning control to user process
2022-01-20 19:12:54,803 INFO    Thread-2  :224864 [wandb_run.py:finish():1457] finishing run amazonproject/Amazon-Reviews-v2/xpfo0wdl
2022-01-20 19:12:54,804 INFO    Thread-2  :224864 [wandb_run.py:_atexit_cleanup():1780] got exitcode: 0
2022-01-20 19:12:54,806 INFO    Thread-2  :224864 [wandb_run.py:_restore():1752] restore
2022-01-20 19:12:54,814 INFO    Thread-2  :224864 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 788
  total_bytes: 788
}

2022-01-20 19:12:54,936 INFO    Thread-2  :224864 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 788
  total_bytes: 788
}

2022-01-20 19:12:55,171 INFO    Thread-2  :224864 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 788
  total_bytes: 788
}

2022-01-20 19:12:55,610 INFO    Thread-2  :224864 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 788
  total_bytes: 2407
}

2022-01-20 19:12:55,712 INFO    Thread-2  :224864 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 788
  total_bytes: 2407
}

2022-01-20 19:12:55,814 INFO    Thread-2  :224864 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 788
  total_bytes: 2407
}

2022-01-20 19:12:55,916 INFO    Thread-2  :224864 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 1648
  total_bytes: 2407
}

2022-01-20 19:12:56,017 INFO    Thread-2  :224864 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 2183
  total_bytes: 2407
}

2022-01-20 19:12:56,119 INFO    Thread-2  :224864 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 2407
  total_bytes: 2407
}

2022-01-20 19:12:56,221 INFO    Thread-2  :224864 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 2407
  total_bytes: 2407
}

2022-01-20 19:12:56,322 INFO    Thread-2  :224864 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 2407
  total_bytes: 2407
}

2022-01-20 19:12:56,424 INFO    Thread-2  :224864 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 2407
  total_bytes: 2407
}

2022-01-20 19:12:56,525 INFO    Thread-2  :224864 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 2407
  total_bytes: 2407
}

2022-01-20 19:12:57,104 INFO    Thread-2  :224864 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 2407
  total_bytes: 2407
}

2022-01-20 19:12:57,579 INFO    Thread-2  :224864 [wandb_run.py:_wait_for_finish():1912] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 2407
  total_bytes: 2407
}
local_info {
}

2022-01-20 19:12:58,708 INFO    Thread-2  :224864 [wandb_run.py:_append_history():2130] rendering history
2022-01-20 19:12:58,709 INFO    Thread-2  :224864 [wandb_run.py:_append_summary():2085] rendering summary
2022-01-20 19:12:58,709 INFO    Thread-2  :224864 [wandb_run.py:_append_files():2180] logging synced files
